Print_A_Bot/controls/utils/system.py

```diff
@@ -0,0 +1,40 @@
+import subprocess
+from logging import getLogger
+from os import path, setpgrp
+
+from django.conf import settings
+
+
+logger = getLogger(__name__)
+
+
+def call_sudo_command(command, new_process=False, **kwargs):
+    """
+    Call django management command with kwargs -- kwargs keys must match command line option names vs var names.
+    """
+    cmd_args = []
+    for k, v in kwargs.iteritems():
+        if type(v) is bool:
+            cmd_args.append('--%s' % k)
+        else:
+            cmd_args.append('--{}={}'.format(k, v))
+    logger.debug('making sudo command call %s %s' % (command, ' '.join(cmd_args)))
+    cmd = ['sudo', 'python', path.join(settings.BASE_DIR, 'manage.py'), command] + cmd_args
+    if new_process:
```

Comment:
normally `subprocess.Popen([cmd])  would be all you need to call a program from python

the extra args are to make the django process calling this method to continue running rather than waiting for exit